xml/docu_styleguide.structure.xml

```diff
@@ -1022,6 +1022,15 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
       apply a proportionally larger maximum window size.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      Size screenshots correctly with the <literal>witdh</literal> attribute to
+      ensure readability and consistency. Screenshots of the whole desktop
+      should use the full width of 100%, whereas shots of individual
+      application windows should use 75%. Small windows such as confirmation
+      dialogs should be 50%.
+     </para>
```

Comment:
Just an idea: what about a list? Sometimes it's a bit hard to absorb all the information, so a list may help here. Something like this:

Size screenshots correctly with the width attribute to ensure readability and consistency. Use:

* 100% for screenshots of the whole desktop,
* 75% for individual application windows, and
* 50% for small windows such as confirmation dialogs.

Reply:
One addition, I would like the authors to have some wiggle room:

* 90-100% - desktop
* 70-80% - app window
* 50-60% - confirmation boxes